sssd_test_framework/roles/ipa.py

```diff
@@ -455,19 +455,28 @@ def add(
     def modify(
         self,
         *,
+        first: str | None = None,
+        last: str | None = None,
         uid: int | None = None,
         gid: int | None = None,
         password: str | None = None,
         home: str | None = None,
         gecos: str | None = None,
         shell: str | None = None,
         user_auth_type: str | list[str] | None = None,
+        idp: str | None = None,
+        idp_user_id: str | None = None,
+        password_expiration: str | None = None,
     ) -> IPAUser:
         """
         Modify existing IPA user.
 
         Parameters that are not set are ignored.
```

Comment:
Redundant means a little more in the Generic class, I think you should remove this comment.

Reply:
removing